inlang/source-code/sdk/src/createMessagesQuery.test.ts

```diff
@@ -2,15 +2,72 @@
 import { describe, it, expect } from "vitest"
 import { createMessagesQuery } from "./createMessagesQuery.js"
 import { createEffect, createRoot, createSignal } from "./reactivity/solid.js"
-import type { Message, Text } from "@inlang/message"
+import { Message, type Text } from "@inlang/message"
 import { createMessage } from "./test-utilities/createMessage.js"
+import { createNodeishMemoryFs } from "@lix-js/fs"
+import type { resolveModules } from "./resolve-modules/resolveModules.js"
+import type { ProjectSettings } from "./versionedInterfaces.js"
+import type { InlangProject } from "./api.js"
 
 const createChangeListener = async (cb: () => void) => createEffect(cb)
 const nextTick = () => new Promise((resolve) => setTimeout(resolve, 0))
 
-describe("create", () => {
-	it("should create a message", () => {
-		const query = createMessagesQuery(() => [])
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const awaitableCreateMessageQuery = async (_messages: () => Message[]) => {
```

Comment:
not usually a fan of naming with prefix like "awaitable" but in this case it clarifies the test - so 👍